src/components/avatars.tsx

```diff
@@ -30,7 +37,11 @@ export function Avatars() {
 	const remainingUsersText = remainingUsers.map( ( { userInfo } ) => userInfo.name ).join( ', ' );
 
 	return visibleUsers.length > 1 ? (
-		<>
+		<div
+			className="vip-real-time-collaboration-avatars-container"
+			onMouseEnter={ () => setIsPopoverVisible( true ) }
+			onMouseLeave={ () => setIsPopoverVisible( false ) }
```

Comment:
In order to be accessible, the popover must be triggerable from a keyboard-focusable element, typically a link, `<button>`, or element with `role="button"`.